hack/e2e.go

```diff
@@ -207,6 +207,11 @@ func run(deploy deployer) error {
 	}
 
 	if *up {
+		// If we tried to bring the Kubemark cluster up, make a courtesy
+		// attempt to bring it down so we're not leaving resources around.
+		if *down {
+			defer xmlWrap("TearDown", deploy.Down)
```

Comment:
There is no need for duplicate step names. It is trivial to change the name of a string literal. Let's address the problem in this PR.